core/src/main/java/org/jdbi/v3/core/Handles.java

```diff
@@ -19,12 +19,14 @@
  * Configuration class for handles.
  */
 public class Handles implements JdbiConfig<Handles> {
-    private boolean forceEndTransactions = true;
```

Comment:
I'm not sure why this is better.  Why are we moving initializers away from the declaration?  It's nice to know the initial value right away, and means you don't have to repeat it in every constructor.

Reply:
I was with you on this (initialize-to-field for clarity), but @qualidafial made an argument for symmetric constructors